### PR TITLE
Nautilus Featured

### PR DESCRIPTION
**Pull request by bgodlin**

### CODE REVIEW

**Review by deverka — APPROVED** [review]

**Comment by deverka** [comment]
@jamesbayly

File: Nautilus/nautilus-starter/project.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  EthereumHandlerKind,
+} from "@subql/types-ethereum";
+
+// Can expand the Datasource processor types via the generic param
+const project: EthereumProject = {
+  specVersion: "1.0.0",
+  version: "0.0.1",
+  name: "nautilus-subql-starter",
+  description:
+    "This project can be use as a starting point for developing your new Ethereum SubQuery project",
```
Comment (jamesbayly) [inline Nautilus/nautilus-starter/project.ts:13]:
```suggestion
    "This project can be use as a starting point for developing your new Nautilus SubQuery project",
```

File: Nautilus/nautilus-starter/project.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    /**
+     * chainId is the EVM Chain ID, for Nautilus this is 22222
+     * https://chainlist.org/chain/22222
+     */
+    chainId: "22222",
+    /**
+     * These endpoint(s) should be public non-pruned archive node
+     * We recommend providing more than one endpoint for improved reliability, performance, and uptime
+     * Public nodes may be rate limited, which can affect indexing speed
+     * When developing your project we suggest getting a private API key
```
Comment (jamesbayly) [inline Nautilus/nautilus-starter/project.ts:37]: This has been updated on the main branch, please use the new formats

File: Nautilus/nautilus-starter/project.ts [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+     */
+    endpoint: ["https://api.nautilus.nautchain.xyz"],
+  },
+  dataSources: [
+    {
+      kind: EthereumDatasourceKind.Runtime,
+      startBlock: 23976397,
```
Comment (jamesbayly) [inline Nautilus/nautilus-starter/project.ts:44]: Why was this chosen? Add it in as a comment

File: Nautilus/nautilus-starter/README.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
+# SubQuery - Example Project for Nautilus
+
+[SubQuery](https://subquery.network) is a fast, flexible, and reliable open-source data indexer that provides you with custom APIs for your web3 project across all of our supported networks. To learn about how to get started with SubQuery, [visit our docs](https://academy.subquery.network).
+
+**This SubQuery project indexes all transfers and approval events for [Ether](https://nautscan.com/mainnet/search/0xB2723928400AE5778f6A3C69D7Ca9e90FC430180) (`0xBDa330Ea8F3005C421C8088e638fBB64fA71b9e0`) on Nautilus Mainnet**
```
Comment (jamesbayly) [inline Nautilus/nautilus-starter/README.md:5]: Wrapped ether?

File: Nautilus/nautilus-starter/docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+      DB_HOST: postgres
+      DB_PORT: 5432
+    volumes:
+      - ./:/app
+    command:
+      - ${SUB_COMMAND:-} # set SUB_COMMAND env variable to "test" to run tests
+      - -f=/app
+      - --db-schema=app
+      - --workers=4
+      - --batch-size=1
```
Comment (jamesbayly) [inline Nautilus/nautilus-starter/docker-compose.yml:39]: Can we leave this is a default of 30
```suggestion
      - --batch-size=30
```

**Review by jamesbayly — CHANGES_REQUESTED** [review]
Theres a few issues, these same things keep coming up

File: Nautilus/nautilus-starter/project.ts [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+  EthereumProject,
+  EthereumDatasourceKind,
+  EthereumHandlerKind,
+} from "@subql/types-ethereum";
+
+// Can expand the Datasource processor types via the generic param
+const project: EthereumProject = {
+  specVersion: "1.0.0",
+  version: "0.0.1",
+  name: "nautilus-subql-starter",
```
Comment (jamesbayly) [inline Nautilus/nautilus-starter/project.ts:11]:
```suggestion
  name: "nautilus-starter",
```

File: Nautilus/nautilus-starter/package.json [hunk 7]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "nautilus-subql-starter",
```
Comment (jamesbayly) [inline Nautilus/nautilus-starter/package.json:2]:
```suggestion
  "name": "nautilus-starter",
```

File: Nautilus/nautilus-starter/LICENSE [hunk 8]
```diff
@@ -0,0 +1,21 @@
+MIT LICENSE
+
+Copyright 2020-2021 SubQuery Pte Ltd authors & contributors
```
Comment (jamesbayly) [inline Nautilus/nautilus-starter/LICENSE:3]:
```suggestion
Copyright 2020-2024 SubQuery Pte Ltd authors & contributors
```

**Review by jamesbayly — APPROVED** [review]

**Pull request merged**